app/controllers/application_controller.rb

```diff
@@ -9,22 +9,30 @@ class ApplicationController < ActionController::Base
     if: -> { ENV["BASIC_AUTH_USERNAME"].present? },
   )
 
-  helper_method :admin_signed_in?, :claim_timeout_in_minutes, :claim_timeout_warning_in_minutes
+  helper_method :current_policy_routing_name, :admin_signed_in?, :admin_timeout_in_minutes, :claim_timeout_in_minutes, :timeout_warning_in_minutes
   before_action :end_expired_admin_sessions
   before_action :end_expired_claim_sessions
   before_action :update_last_seen_at
 
   private
 
+  def current_policy_routing_name
+    params[:policy]
```

Comment:
We've already get plenty test coverage for the behaviour that this override delivers – remove the override in `PartOfClaimJourney` and a whole heap of specs will fail. A test to verify that this specific private method has been overridden would not be a great test. It would be testing the **implementation**, and not the intended **behaviour**, and would couple the test to the specifics of the implementation.